adjust_restart_for_new_land_cover/adjust_restart_for_new_land_cover.py

```diff
@@ -0,0 +1,396 @@
+    """Modify the input search mask to only include the current grid cell at
+    Coord."""
+
+    Lon, Lat = Coord
+    SearchMask[Lat, Lon] = True
+
+def modify_mask_for_nearest(SearchMask, SearchRadius, Coord):
+    """Modify the input search mask to only include the specified 'radius' of
+    points around the given Coord. In this instance, radius is just a block of
+    points with index +-SearchRadius around the original Coord."""
+    
+    # Destructure for more readable usage later
+    Lon, Lat = Coord
+    nLat, nLon = SearchMask.shape
+
+    # Create the set of coordinates to unmask
+    SearchInds = numpy.arange(-SearchRadius, SearchRadius+1)
+
+    # Clip the latitude indices, since they aren't periodic
+    LatInds = numpy.clip(Lat + SearchInds, 0, nLat-1)
+
+    # Apply mod to longitude indices, since they are periodic
+    LonInds = numpy.mod(Lon + SearchInds, nLon)
+
+    # Modify the mask in place
+    for LatSearch in LatInds:
+        for LonSearch in LonInds:
+            SearchMask[LatSearch, LonSearch] = True
+
+def modify_mask_for_latitude_band(SearchMask, LatitudeBand, Coord):
+    """Modify the input search mask to only include the specifed band of
+    latitudes around the given Coord."""
+
+    # Destructure for more readable usage later
+    Lon, Lat = Coord
+    nLat, nLon = SearchMask.shape
+
+    # Create the set of coordinates to unmask
+    SearchInds = numpy.arange(-LatitudeBand, LatitudeBand+1)
+
+    # Clip the latitude indices, since they aren't periodic
+    LatInds = numpy.clip(Lat + SearchInds, 0, nLat-1)
+
+    # Modify the mask in place
+    for LatSearch in LatInds:
+        SearchMask[LatSearch, :] = True
+
+def modify_mask_for_global(SearchMask, Global, Coord):
+    """Modify the input search mask to include the whole globe. Include
+    arguments for consistency with the other masking methods."""
+
+    # Nothing interesing to do here
+    SearchMask[:] = True
+
+def find_active_tiles(
+        SearchMask,
+        InputVegetation,
+        VegetationMapping,
+        ):
+    """Using the supplied search mask, generate a set of masks which are an OR
+    of the original search mask and a mask describing the active tiles, defined
+    as a tile with an area fraction of greater than 1e-6. Generates a mask for
+    each of the vegetation types used to construct the new vegetation type."""
+
+    # Make it a list, because there are instances where more than 1 input
+    # vegetation type maps to a single output vegetation type
+    ActiveTileMasks = []
+
+    for VegType in VegetationMapping:
+        # The search mask should then be a logical and of:
+        # * The land points (where the vegetation is non nan)
+        # * The supplied search mask
+        # * The tiles which have a vegetation type greater than 0.0
+        # Note that the minimum area threshold is chosen carefully to be
+        # greater than 1e-6, which is the value used for tiles that will become
```

Comment:
```suggestion
        # greater than 0.0, which is the value used for tiles that will become
```
Should this comment also be swapped to 0.0?

Reply:
Yep, fixed.